packages/@aws-cdk/aws-lambda/lib/layers.ts

```diff
@@ -0,0 +1,237 @@
+    }
+
+    new cdk.Resource(this, grantId, {
+      type: 'AWS::Lambda::LayerVersionPermission',
+      properties: {
+        Action: 'lambda:GetLayerVersion',
+        LayerVersionArn: this.layerVersionArn,
+        Principal: grantee.accountId,
+        OrganizationId: grantee.organizationId,
+      }
+    });
+    return this;
+  }
+
+  /**
+   * Exports this layer for use in another Stack. The resulting object can be passed to the ``LayerVersion.import``
+   * function to obtain an ``ILayerVersion`` in the user stack.
+   */
+  public export(): ImportedLayerVersionProps {
+    return {
+      layerVersionArn: new cdk.Output(this, 'LayerVersionArn', { value: this.layerVersionArn }).makeImportValue().toString(),
+      compatibleRuntimes: this.compatibleRuntimes,
+    };
+  }
+}
+
+/**
+ * Identification of an account (or organization) that is allowed to access a Lambda Layer Version.
+ */
+export interface LayerVersionUsageGrantee {
+  /**
+   * The AWS Account id of the account that is authorized to use a Lambda Layer Version. The wild-card ``'*'`` can be
+   * used to grant access to "any" account (or any account in an organization when ``organizationId`` is specified).
+   */
+  accountId: string;
+
+  /**
+   * The ID of the AWS Organization to hwich the grant is restricted.
+   *
+   * Can only be specified if ``accountId`` is ``'*'``
+   */
+  organizationId?: string;
+}
+
+/**
+ * Properties necessary to import a LayerVersion.
+ */
+export interface ImportedLayerVersionProps {
+  /**
+   * The ARN of the LayerVersion.
+   */
+  layerVersionArn: string;
+
+  /**
+   * The list of compatible runtimes with this Layer.
+   */
+  compatibleRuntimes?: Runtime[];
+}
+
+/**
+ * Defines a new Lambda Layer version.
+ */
+export class LayerVersion extends LayerVersionBase {
+  /**
+   * Imports a Layer that has been defined externally.
+   *
+   * @param parent the parent Construct that will use the imported layer.
+   * @param id     the id of the imported layer in the construct tree.
+   * @param props  the properties of the imported layer.
+   */
+  public static import(parent: cdk.Construct, id: string, props: ImportedLayerVersionProps): ILayerVersion {
+    return new ImportedLayerVersion(parent, id, props);
+  }
+
+  public readonly layerVersionArn: string;
+  public readonly compatibleRuntimes?: Runtime[];
+
+  constructor(parent: cdk.Construct, id: string, props: LayerVersionProps) {
+    super(parent, id);
+    if (props.compatibleRuntimes && props.compatibleRuntimes.length === 0) {
+      throw new Error('Attempted to define a Lambda layer that supports no runtime!');
+    }
+    if (props.code.isInline) {
+      throw new Error('Lambda layers cannot be created from inline code');
+    }
+    // Allow usage of the code in this context...
+    props.code.bind(this);
+
+    const resource = new cdk.Resource(this, 'Resource', {
```

Comment:
Sooo... no proper L1s for this yet? How come?

Reply:
Was missing from the CFN Spec when I initially did that.